CaseyShimata/ACHLibrary
# Add regex test to the ACHFieldRule class; Fixed some control statement

Testing instructions: 

run the REST API from postman as you did in previous tests
look for new validation tests and observe their results. 

Note: I will be cleaning up the results more in a separate merge.
